test/language/expressions/tagged-template/invalid-escape-sequences.js

```diff
@@ -0,0 +1,70 @@
+// Copyright (C) 2016 Tim Disney. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+/*---
+description: Invalid unicode escape sequence in tagged template literals are allowed
+es6id: 11.8.6
```

Comment:
es6id is a strict reference to the es6 release. Anything after ES6 should use esid tag only.

If the section is undefined, we conventionally use `esid: pending`